### ISSUE TITLE
&quot;No module named marey_graph&quot; when running schedule_viewer from /usr/bin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#24]
```
When trying to execute schedule_viewer.py under Fedora, no matter if I
execute after running setup.py, it always report an import error:

Traceback (most recent call last):
  File "/usr/bin/schedule_viewer.py", line 26, in ?
    import marey_graph
ImportError: No module named marey_graph

What version of the product are you using? On what operating system?
I am using transitfeed-1.0.7. My operating system is version 2.6.20-1.2944.fc6

```

Original issue reported on code.google.com by `juan...@gmail.com` on 22 May 2007 at 11:02

**Comment by GoogleCodeExporter** [comment]
```
Anyway, it seems to work if I comment this line :S
```

Original comment by `juan...@gmail.com` on 22 May 2007 at 11:57

**Comment by GoogleCodeExporter** [comment]
```
Good report. I had not tested running schedule_viewer after using
setup.py install and reproduced the problem.

You will probably be missing the trip details at the bottom but it should 
otherwise
work. Try running ./schedule_viewer.py directly from the source directory.

Looks like you were in the source directory anyway because the data files are 
put in
/usr/schedule_viewer_files(!) and /usr/bin/schedule_viewer.py reads relative to 
cwd.
See issue 19.

```

Original comment by `tom.brow...@gmail.com` on 22 May 2007 at 1:43
- Changed title: **&quot;No module named marey_graph&quot; when running schedule_viewer from /usr/bin**
- Added labels: **App-ScheduleViewer**, **Language-Python**

**Comment by GoogleCodeExporter** [comment]
Original comment by `joe.hugh...@gmail.com` on 22 May 2007 at 2:31

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed by tom.brown.code-setup.py-fixes in transitfeed 1.0.8
```

Original comment by `tom.brow...@gmail.com` on 30 May 2007 at 7:41
- Changed state: **Fixed**
